hydra-main/web/spawn2/build/1.main.js

```diff
@@ -23542,11 +23542,17 @@ webpackJsonp([1],Array(215).concat([
 	                checkbox.attr("class","checkbox checked-all");
 	                checkbox.parent().data("state","all");
 	                this.views.selectable.find("button.btn-hide-zero").removeClass("disabled");
+	                this.views.selectable.find("button.btn-show-one").addClass("disabled");
 	            }
 	            else if(keys.length>0){
-	                checkbox.attr("class","checkbox checked-partial");
-	                checkbox.parent().data("state","partial");
-	                this.views.selectable.find("button.btn-hide-zero").removeClass("disabled");
+	                if(keys.length == 1) {
+	                    this.views.selectable.find("button.btn-show-one").removeClass("disabled");
+	                } else {
+	                    checkbox.attr("class","checkbox checked-partial");
+	                    checkbox.parent().data("state","partial");
+	                    this.views.selectable.find("button.btn-hide-zero").removeClass("disabled");
+	                    this.views.selectable.find("button.btn-show-one").addClass("disabled");
+	                }
```

Comment:
It's slightly better to flatten this. Instead of:
```
else if (keys.length>0) {
  if (keys.length == 1) {
    ...
  } else {
    ...
  }
}
```

just do:
```
else if (keys.length==1) {
  ...
} else if (keys.length>1) {
  ...
}
```